arangod/Cluster/ClusterMethods.cpp

```diff
@@ -1168,10 +1168,11 @@ Result selectivityEstimatesOnCoordinator(ClusterFeature& feature, std::string co
     if (tid != 0) {
       headers.try_emplace(StaticStrings::TransactionId, std::to_string(tid));
     }
-
+    
+    reqOpts.param("collection", StringUtils::urlEncode(p.first));
```

Comment:
won't this double-urlencode the parameter?

Reply:
good catch